e2e/_suites/ingest-manager/features/fleet_mode_agent.feature

```diff
@@ -21,8 +21,8 @@ Scenario: Stopping the agent stops backend processes
   Then the "filebeat" process is in the "stopped" state on the host
     And the "metricbeat" process is in the "stopped" state on the host
 
-@restart-agent
-Scenario Outline: Restarting a <os> agent restarts backend processes
+@restart-host
+Scenario Outline: Restarting the <os> host with persistent agent restarts backend processes
```

Comment:
@EricDavisX what I'm not sure about, it's the need of adding `persistent agent`: at the moment we moved from TAR to a package, it will always use persistent ones. Unless we create dedicated scenarios for TAR, I think it's something we should move to the Feature level:

```gherkin
Feature: Fleet Mode Agent
  Scenarios for the Agent in Fleet mode connecting to Ingest Manager application, using a persistent installer
```